test/Microsoft.Azure.ServiceBus.UnitTests/QueueSessionTests.cs

```diff
@@ -188,6 +189,28 @@ public sealed class QueueSessionTests
             }
         }
 
+        [Theory]
+        [MemberData(nameof(TestPermutations))]
+        [DisplayTestMethodName]
+        async Task ServerWaitTimeoutTest(string queueName, int messageCount = 1)
```

Comment:
@SeanFeldman - Were you still interested in submitting a PR for the unit test name refactoring?